src/store/modules/display/getters.ts

```diff
@@ -105,6 +106,11 @@ export const allMenuList: Getter<DisplayState, any> = (state, getters, rootState
     return _allGnbMenuList;
 };
 
-export const GNBMenuList: Getter<DisplayState, any> = (state, getters): DisplayMenu[] => getters.allMenuList.filter((d) => !d.hideOnGNB);
+export const GNBMenuList: Getter<DisplayState, any> = (state, getters): DisplayMenu[] => getters.allMenuList.filter((d) => {
+    if (d.id === MENU_ID.DASHBOARDS) {
+        return (config.get('DASHBOARD_ENABLED'));
```

Comment:
suggestion: what about checking hideOnGNB also?

```suggestion
        return !d.hideOnGNB && !!config.get('DASHBOARD_ENABLED');
```